test/Benchmarks/src/Text/Regex_Bench.enso

```diff
@@ -1,20 +1,37 @@
 from Standard.Base import all
+from Standard.Base.Runtime import assert
 
 import Standard.Base.Data.Text.Regex.Internal.Replacer.Replacer
 
 from Standard.Test import Bench, Faker
 
 
 type Data
-    Value ~two_nums ~just_two_nums ~four_nums ~a_regex ~regexes ~replacers ~one_num_regex ~two_nums_regex
+    Value
+        ~two_nums
+        ~just_two_nums
+        ~four_nums
+        ~a_regex
+        ~regexes
+        ~replacers
+        ~one_num_regex
+        ~two_nums_regex
 
     create =
         faker = Faker.new
         a_regex = Regex.compile "(\d)ab(\d)"
-        Data.Value (build_two_nums faker) (build_just_two_nums faker) (build_four_nums faker) a_regex (build_regexes faker) (build_replacers faker) (Regex.compile "(\d+)") (Regex.compile "(\d+)[a-z]+(\d+)")
-
-list_count = 1000000
-regex_count = 10000
+        Data.Value
+            <| (build_two_nums faker)
+            <| (build_just_two_nums faker)
+            <| (build_four_nums faker)
+            <| a_regex
+            <| (build_regexes faker)
+            <| (build_replacers faker)
+            <| (Regex.compile "(\d+)")
+            <| (Regex.compile "(\d+)[a-z]+(\d+)")
```

Comment:
Och, óóó! That's way nicer! Honestly, up until yesterday I wasn't even aware we support such a nice syntax! I just confused this with @kazcw's _multiline constructor definition_ and then wanted to find a way out without admitting I got confused...

Reply:
This would also work without the parens. In an operator block, each operator takes the rest of the line as its RHS.

Reply:
Removed parens